util/dvsim/Modes.py

```diff
@@ -301,6 +302,7 @@ def __init__(self, rdict):
         self.uvm_test = ""
         self.uvm_test_seq = ""
         self.build_mode = ""
+        self.timeout_mins = 60
```

Comment:
Same as above comment:
```suggestion
        self.run_timeout_mins = 60
```